step-cli/step-cli-core/src/main/java/step/cli/parameters/ApDeployParameters.java

```diff
@@ -0,0 +1,45 @@
+package step.cli.parameters;
+
+public class ApDeployParameters extends ApParameters<ApDeployParameters> {
+    private Boolean async;
```

Comment:
Why are the boolean values objects instead of primitive booleans? (this might be intentional to allow/set by default an "undefined" value, but I thought I'd ask...)